pkg/services/sqlstore/org_users.go

```diff
@@ -117,7 +116,7 @@ func (ss *SQLStore) GetOrgUsers(ctx context.Context, query *models.GetOrgUsersQu
 	// service accounts table in the modelling
 	whereConditions = append(whereConditions, fmt.Sprintf("%s.is_service_account = %t", x.Dialect().Quote("user"), query.IsServiceAccount))
 
-	if ss.Cfg.FeatureToggles["accesscontrol"] {
+	if ss.Cfg.FeatureToggles["accesscontrol"] && query.User != nil {
```

Comment:
hmm, interesting fix here. So we get a query which does not have a user I see.
But this is also related to "access control" not checking for a user right